### ISSUE TITLE
NS1002 - doesnt detect non-virtual calls from base classe

### ISSUE THREAD

**Issue by tpodolak** [nsubstitute/NSubstitute.Analyzers#89]
Repro
````c#
using NSubstitute;
namespace MyNamespace
{
    public class AbstractFoo
    {
        public int Bar()
        {
            return 1;
        }
    }

    public class Foo : AbstractFoo
    {
    }
    
    public class FooTests
    {
        public void Test()
        {
            var foo = new Foo();

            foo.When(xx => xx.Bar()); // xx.Bar() is not higlighted
        }
    }
}
````
